modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/search/CPInstanceIndexer.java

```diff
@@ -71,6 +71,8 @@ public CPInstanceIndexer() {
 			Field.COMPANY_ID, Field.ENTRY_CLASS_NAME, Field.ENTRY_CLASS_PK,
 			Field.GROUP_ID, Field.MODIFIED_DATE, Field.SCOPE_GROUP_ID,
 			CPField.SKU, Field.UID);
+		setFilterSearch(true);
+		setPermissionAware(true);
```

Comment:
why true here and false in the definition?

Reply:
For definitions we are doing filtering based off the catalog/channel already, so it will handle that end of it, I had to turn off permissions so that sites that allow guest like speedwell will still display the products on the storefront. For cpInstance, we still need it since we were just returning all cpInstances in the admin page.